server/filestore_test.go

```diff
@@ -8296,3 +8296,190 @@ func TestFileStoreNumPendingMulti(t *testing.T) {
 	}
 	require_Equal(t, total, checkTotal)
 }
+
+func TestFileStoreMessageTTL(t *testing.T) {
+	fs, err := newFileStore(
+		FileStoreConfig{StoreDir: t.TempDir(), EnforceTTLs: true},
```

Comment:
Can we not inherit enforeTTLS from the stream config?

Reply:
In `stream.go` when creating the store for streams, we do indeed set it from the `AllowMsgTTLs` value from the stream config. In the filestore-specific tests, like here, we set it to true directly.

Reply:
Since we pass in a stream config in the tests we could do it there, or you feel we should keep the redundancy and have it in two places?

Reply:
Not attached to the duplication at all, will remove it and use the passed-in `StreamConfig` one.